apps/cms.c

```diff
@@ -276,6 +274,7 @@ int cms_main(int argc, char **argv)
     CMS_ReceiptRequest *rr = NULL;
     ENGINE *e = NULL;
     EVP_PKEY *key = NULL;
+    char *wrapname = NULL;
```

Comment:
The changes from here down seem not related to this PR..

Reply:
Sigh.  I tried to split this off from #14182 and did not do a good job.   fixup commit pushed.